src/main/java/com/shrug/GUI/GUI.java

```diff
@@ -408,4 +416,20 @@ public void helpDialog(ActionEvent event) {
         "Help",
         JOptionPane.PLAIN_MESSAGE);
   }
+
+  public void ProcessButtonPressExportImage (ActionEvent event) {
+    try {
+      BufferedImage image = mxCellRenderer.createBufferedImage(jgxAdapter, null, 1, Color.WHITE, true, null);

```

Comment:
call `getInputDialogBox ()` to get a filename from the user.
Could look something like `String filename = getInputDialogBox ("Export", "Export to PNG", "Enter a .png file")`